src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/TreeNodeDefinition.xml

```diff
@@ -209,6 +209,8 @@
     <Filters>
       <Filter TypeToReverse="SqlIndex" Property="IndexKeyType" Type="Enum" ValidFor="Sql2016|Sql2017|AzureV12">
         <Value>IndexKeyType.None</Value>
+        <Value>IndexKeyType.DriPrimaryKey</Value>
```

Comment:
Just checking - FKs show up with this change too, correct?

Reply:
FK should only show up under key (based on ssms) and this change doesn't affect that 

---
In reply to: [154440157](https://github.com/Microsoft/sqltoolsservice/pull/557#discussion_r154440157) [](ancestors = 154440157)